plugins/CoreConsole/Commands/GenerateTest.php

```diff
@@ -193,4 +194,86 @@ protected function getTestFilesWhitelist($testType)
             '/tests/Unit/SimpleTest.php'
         );
     }
-}
+
+    private function addTravisJobsIfRequired(OutputInterface $output, $testType, $pluginName)
+    {
+        $travisYmlFile = PIWIK_INCLUDE_PATH . '/plugins/' . $pluginName . '/.travis.yml';
+        if (!file_exists($travisYmlFile)) {
+            $output->writeln("Plugin '$pluginName' has no .travis.yml file, skipping travis job addition.");
```

Comment:
Can we create `travis-yml` in this case or at least show the command to generate it? Also we could explain that one might has to set `artifacts-pass` etc.